### PR TITLE
Update upheno.md

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
add publications
address https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1826

### CODE REVIEW

File: ontology/upheno.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ products:
     title: "uPheno 2"
     description: "The new version of uPheno, along with species independent phenotypes amd additional phenotype relations. The ontology is still in Beta status, but we recommend users to migrate their infrastructures to uPheno 2 as uPheno 1 is no longer actively maintained."
     page: https://github.com/obophenotype/upheno-dev
+publications: 
+   - id: https://zenodo.org/record/2382757#.YmClG9PMJ4E
```
Comment (cthoyt) [inline ontology/upheno.md:31]:
```suggestion
   - id: https://zenodo.org/record/2382757
 ```

File: ontology/upheno.md [hunk 2]
```diff
@@ -27,6 +27,11 @@ products:
     title: "uPheno 2"
     description: "The new version of uPheno, along with species independent phenotypes amd additional phenotype relations. The ontology is still in Beta status, but we recommend users to migrate their infrastructures to uPheno 2 as uPheno 1 is no longer actively maintained."
     page: https://github.com/obophenotype/upheno-dev
+publications: 
+   - id: https://zenodo.org/record/2382757#.YmClG9PMJ4E 
+     title: "Phenotype Ontologies Traversing All The Organisms (POTATO) workshop aims to reconcile logical definitions across species" 
+   - id: https://zenodo.org/record/3352149#.YmClWdPMJ4F
```
Comment (cthoyt) [inline ontology/upheno.md:33]:
```suggestion
   - id: https://zenodo.org/record/3352149
```

**Comment by cthoyt** [comment]
Blocked by #1855

**Comment by cthoyt** [comment]
indendation needs fixing

**Review by matentzn — APPROVED** [review]

**Pull request merged**